projects/blindex/index.js

```diff
@@ -0,0 +1,207 @@
+const sdk = require("@defillama/sdk");
+const abi = require("../helper/abis/blindex.json");
+const { calculateUniTvl } = require("../helper/calculateUniTvl.js");
+
+//-------------------------------------------------------------------------------------------------------------
+// How to add a new chain?
```

Comment:
maybe it's better to have this text in the readme file?

Reply:
It's not our readme file.. we need to merge this code back to their repo, I just forked it.

The problem is that in their SDK I think they didn't think anyone would want to test something on RSK so they didn't add it by default haha

What we can do is open a PR to their SDK and add rsk there as well.
That will help any future RSK protocols testing on RSK.

WDYT?

Reply:
we have a "few" things that are more important to do :)
so we can keep it for another time

Reply:
Haha I think that would be the best use of my time ;)